src/hotspot/share/oops/symbolHandle.hpp

```diff
@@ -43,16 +44,30 @@
 // probe() and lookup_only() will increment the refcount if symbol is found.
 template <bool TEMP>
 class SymbolHandleBase : public StackObj {
+  static Symbol* volatile _cleanup_delay_queue[];
```

Comment:
This looks off. Should be `static Symbol** volatile _cleanup_delay_queue`, maybe?

Reply:
Nevermind, that's not doable here.